application/org.openjdk.jmc.ui/src/main/java/org/openjdk/jmc/ui/dial/Dial.java

```diff
@@ -64,8 +64,26 @@ public class Dial extends Composite {
 	 *            a widget which will be the parent of the new widget
 	 * @param toolkit
 	 *            the form toolkit in use
+	 * @param dialConfiguration
+	 *            the dial configuration to use
 	 */
 	public Dial(Composite parent, FormToolkit formToolkit, DialConfiguration dialConfiguration) {
```

Comment:
Looking at the usage of the Dial and DialViewer constructors, should the old ones be removed? I don't see them being used anywhere